mcs/class/System.Web/System.Web.Util/HttpEncoder.cs

```diff
@@ -167,6 +167,12 @@ static HttpEncoder GetCustomEncoderFromConfig ()
 			return defaultEncoder.Value;
 #else
 			var cfg = HttpRuntime.Section;
+			// The unityjit profile is only built once for desktop and mobile.
+			// When the jit profile is running on a mobile platform (aka Android) this returns null.
+			// We will simply return the default encoder.
+			if(cfg == null)
```

Comment:
Should we add a runtime platform check here?

Reply:
We could and I considered it. I followed the logic below and it appears that most paths will return default on line 179. The code is looking for a type that extends HttpEncode to create. It seems safe to assume this on all platforms.